skills/docx/scripts/__init__.py

```
@@ -0,0 +1 @@
+# Make scripts directory a package for relative imports in tests
```

Ruff (0.14.3)

The file is executable but no shebang is present

(EXE002)